README.md

```diff
@@ -109,6 +109,13 @@ ks param set kubeflow-core usageId $(uuidgen)
 ks apply default -c kubeflow-core
 ```
 
+If the deployment is on a single Kubernetes node (e.g. on an on-premise machine) without access to a loadbalancer, then we need these two lines to access the Jupyter notebook: 
```

Comment:
Can you provide more explanation of what this is doing and when to do it?  This is exposing the service on every node to allow external communication to the K8s service. But this will only work if users are able to connect to the VM and node. 

You would also probably want to update the instructions to explain how users should leverage this to actually connect to jupyter hub; i.e.
- identify ip address of node
- connect to JupyterHub via http://node:port

But this is arguably an incomplete solution because Kubeflow consists of many services and they would have to expose each one.

Have you seen #11 to get a set of  Kubeflow's networking architecture; any idea what the correct setup might be for you platform?

Reply:
Thx for the feedback .... I agree its not a complete solution since there are so many options. While a proxy or a VPN might be a proper solution, this README.md could be kept simple (optionally) to help the first time user who wishes to try kubeflow on a local server (with disclaimers).